modules/t/sqlHelper.t

```diff
@@ -27,6 +27,8 @@ use Bio::EnsEMBL::Test::MultiTestDB;
 use Bio::EnsEMBL::Test::TestUtils;
 use Bio::EnsEMBL::Utils::SqlHelper;
 
+use Bio::EnsEMBL::Hive::DBSQL::DBConnection;
```

Comment:
What happens if a user wants to run this test without having a local checkout of Hive? Would it be possible to run this test only if sHive is available and not fail if it is not?
That way, Travis should still run fine and run this test, but we are not requiring external users to install eHive to be able to run the test suite

Reply:
Hi mag,
You mean something like this https://github.com/Ensembl/ensembl-hive/blob/version/2.5/t/05.runnabledb/fastafactory.t#L33-L37 ?

Reply:
Exactly, or https://github.com/Ensembl/ensembl/blob/release/93/modules/t/utilsIo.t#L145